src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  selectedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    selectedKey: 'Nothing was pressed yet',
```

Comment:
It's better set `null` if there is no pressed key.
And message should be rendered based on this nullable value